pkg/controller/adam/adam.go

```diff
@@ -99,6 +119,38 @@ func (adam *Ctx) getInfoRedisStream(devUUID uuid.UUID) (dir string) {
 	return fmt.Sprintf("INFO_EVE_%s", devUUID.String())
 }
 
+//getLogsRedisStreamCache return logs stream for devUUID for caching in redis
```

Comment:
Please refactor it to Rediscashing.go or something like that. Very misleading to have it in Adam